examples/rabbitmq-client-constraint-validation/rabbitmq_client_constraint_validation.md

```diff
@@ -1,6 +1,6 @@
 # RabbitMQ client - Constraint validation
 
-This example shows how the payload is validated related to the constraints added to the payload record. When a payload is not valid, an error of type `rabbitmq:PayloadValidationError` is returned.
+The RabbitMQ client supports validating a payload by the constraints added to the related message record. A RabbitMQ client can be created by passing the host and port of the RabbitMQ broker. To consume messages, use the `consumePayload` or `consumeMessage` method, which requires the queue name as the argument. When a message with a constraint is received from the RabbitMQ server, it is validated internally. If the validation fails, an error of the `rabbitmq:PayloadValidationError` type gets returned. Payload validation using constraints can be used when the message content needs to be implicitly validated upon receiving from the server.
```

Comment:
Let's make this consistent with the other constraint validation BBEs.

Reply:
Fixed in 48acff4cea36f7b10129081400d9fcc43d345181 please review